db_master_deploy/dml_trigger.sh

```diff
@@ -91,16 +91,18 @@ get_service_search_sphinx_version() {
     # located at the provided URL
     ########################################
     local staging=$1
-    local url="https://sys-api3.${staging}.bgdi.ch/rest/services/ech/SearchServer/info"
+    local rdm_number=$(( ( RANDOM % 100 ) ))
+    local url="https://sys-api3.${staging}.bgdi.ch/rest/services/ech/SearchServer/info?rdm=${rdm_number}"
```

Comment:
I would either use the `X-E2E-Tests` and `If-None-Match` header for by-passing cache.